### DISCUSSION TITLE
apt install

### DISCUSSION THREAD

**Discussion by TxcToxic** [fail2ban/fail2ban#3585] [Q&A] [answered]
why isn't the newest version of fail2ban available via apt install fail2ban

**Comment by sebres** [comment, marked as answer]
Fully answered here - #2308.

Basically distro package maintainers are responsible for distro packaging. But it can be also policies of some distro that not allow to upgrade to major release.
Anyway - there is also a wiki article for this - [How to install or upgrade fail2ban manually](https://github.com/fail2ban/fail2ban/wiki/How-to-install-or-upgrade-fail2ban-manually).
